### DISCUSSION TITLE
[Samsung Galaxy S23 Ultra] In-game cursor does not move according to s-pen's hover cursor

### DISCUSSION THREAD

**Discussion by Aragami1408** [ppy/osu#28798] [Q&A]
Hello everyone, I just updated osu!lazer to the latest version of osu! lazer.

However, there is an issue that using s-pen hover style doesn't work for me anymore since the cursor only move when I touch the screen.

Hopefully there's an option in the game setting to enable hovering mode.

**Comment by 4310V343k** [comment]
Xiaomi Pad 6 with its stylus, also no hover inputs. Only touching the screen works. I'll try to downgrade the osu! version later today

**Comment by 4310V343k** [comment]
2024.625.2 - doesn't work
2024.521.2 - works
2024.519.1 - doesn't work
2024.412.1 - works
